jobs/pull-kubernetes-federation-e2e-gce-gci.sh

```diff
@@ -47,7 +47,7 @@ export KUBE_FASTBUILD=true
 ./hack/jenkins/build.sh
 
 # Push federation images to GCS.
-./build-tools/push-federation-images.sh
+./build/push-federation-images.sh
```

Comment:
`hack/jenkins/build.sh` calls into `kubernetes/release/push-build.sh`, which [calls back to `push-federation-images.sh`](https://github.com/kubernetes/release/blob/master/push-build.sh#L185)